packages/react/src/auto/polaris/submit/PolarisAutoSubmit.tsx

```diff
@@ -15,11 +16,13 @@ import React from "react";
 export const PolarisAutoSubmit = (
   props: {
     children?: ReactNode;
-    isSubmitting?: boolean;
```

Comment:
I think we should continue to export this for backwards compatibility, and fallback to the context value if it is not provided

Reply:
Good call. Updated to keep the original prop around and fallback to the context value